src/pages/login/index.tsx

```diff
@@ -13,27 +13,25 @@ interface Value {
 }
 
 const Login = () => {
+  const [error, setError] = useState();
+  const [loading, setLoading] = useState(false);
```

Comment:
Suggest to join this 2 in one state

for example:
`type Status = "pending" | "success" | "error";

...

const [status, setStatus] = useState<Status>("pending");`

Reply:
If you want to keep error message, you could use an [union](https://www.typescriptlang.org/docs/handbook/unions-and-intersections.html#discriminating-unions)